src/ussd_higherhealth_healthcheck.js

```diff
@@ -1150,6 +1150,10 @@ go.app = (function () {
           province: answers.state_province,
           city: answers.state_city,
           age: answers.state_age,
+          university: answers.state_university,
+          university_other: answers.state_university_other,
+          campus: answers.state_campus,
+          campus_other: answers.state_campus_other,
```

Comment:
These aren't valid fields in the API, so this request will fail.

I think we'll need to put this in the `data` field, see https://github.com/praekeltfoundation/higher-health-django/blob/develop/higher_health/tasks.py#L63-L88

Reply:
You mean that the API endpoint isn't expecting these values?

Reply:
Yes, those aren't fields in the API spec, because they're specific to HigherHealth, they're not applicable to a generic HealthCheck.

Reply:
@rudigiesler addressed issue on https://github.com/praekeltfoundation/higher-health-django/pull/109

Reply:
I think there's a misunderstanding here. That PR doesn't solve this issue at all.
The code in `higher-health-django` is just an example of what a request to the API could look like, that's not where the API is defined at all.
And we don't want to be changing the API definition for this, the university and campus aren't generic enough to add to the API. We should instead be placing this data in the `data` field, similar to how it is done in the task in `higher-health-django`

Reply:
@rudigiesler I have since updated the api call and amended the tests

Reply:
I still don't think that this is quite right. I think the additional data should be in the `data` field, since these aren't fields that are present in the API, the API will return a 400 error status if we attempt this. But there's no validation on the `data` field, as long as it's valid JSON. ie:
```
city: answers.state_city,
age: answers.state_age,
data: {
  university: {
    name: answers.state_university
  },
  ...
}
```